addons/replacements/CfgVehicles.hpp

```diff
@@ -0,0 +1,39 @@
+class CfgEditorCategories {
+    class CLASS(Replacement_Stuff) {
+        displayName = "GT Armory Replacements";
+    };
+};
+
+class CfgEditorSubcategories {
+    class CLASS(Replacement_Items) {
+        displayName = "Replacement Items";
+    };
+};
+
+class CfgVehicles {
+    class Box_AAF_Uniforms_F;
+    class CLASS(Replacement_Box): Box_AAF_Uniforms_F {
+        displayName = "Replacement Testing Box";
```

Comment:
Why not leave that always? Can make it scope 1 so you can spawn it only with script.